tests/frontend/test_kibana_app/cypress/integration/features/management/settings/logs.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Read Kibana logs
+
+  As a kibana user
+  I want to check the logs
+  in order to see information about the system
```

Comment:
`about the app logging` (?)

Reply:
I didn't want to write something like that because it sounds a little redundant... 
but, I think it is correct to say

  As a Kibana user
  I want to check the app logs
  in order to see information about the app logging
  
or I can just say:
  As a Kibana user
  I want to check the app logs
  in order to validate the correct display of logs in the page.
  
or something like that